packages/expo-modules-core/ios/Core/Views/SwiftUI/SwiftUIHostingView.swift

```diff
@@ -21,6 +21,10 @@ extension ExpoSwiftUI {
      It's an environment object that is observed by the content view.
      */
     private let props: Props
+    /**
+     Additiional utilities for controlling shadow node behavior.
+     */
+    private let utils: ViewUtils = ViewUtils()
```

Comment:
Please rename the `ViewUtils`, in my opinion that name is too broad. Maybe we can call it just a `ShadowNode`?